data-raw/_helper.R

```diff
@@ -0,0 +1,24 @@
+library(here)
+
+internal_data_path <- here("R/sysdata.rda")
+
+save_to_sysdata <- function(obj, obj_name) {
+  # Create a new environment for storing sysdata contents
+  sysdata_env <- new.env(hash = FALSE)
+
+  # Load current internal data into this new environment
```

Comment:
`archive_cases_dv_subset_dt` seems to both have a `.rda` file and appear in sysdata.  Can we delete:
```
data/archive_cases_dv_subset_dt.rda
```
?

Reply:
Yeah, should be able to

Reply:
Actually, I don't see that file. I think it's local to you.